fastai/vision/gan.py

```diff
@@ -121,23 +160,33 @@ def forward(self, output, target):
 
 # Cell
 def accuracy_thresh_expand(y_pred, y_true, thresh=0.5, sigmoid=True):
```

Comment:
```suggestion
def accuracy_thresh_expand(y_pred:Tensor, y_true:Tensor, thresh:float=0.5, sigmoid:bool=True):
```

May also want to docment this

Reply:
thanks!